vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/component/ExperimentInfoEditEvent.java

```diff
@@ -0,0 +1,22 @@
+package life.qbic.datamanager.views.projects.project.experiments.experiment.component;
+
+import com.vaadin.flow.component.ComponentEvent;
+import java.io.Serial;
+
+/**
+ * <b>Experiment Info Edit Event</b>
+ *
+ * <p>Indicates that the user wants to edit experiment information</p>
```

Comment:
Is this event specific for the sample origin or will it be reused for all components within the experiment page?

Reply:
Only for the component.

Reply:
:+1: